pipelines/services/resource.go

```diff
@@ -0,0 +1,45 @@
+package services
+
+import (
+	"encoding/json"
+	"github.com/jfrog/jfrog-client-go/http/jfroghttpclient"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/io/httputils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"net/http"
+)
+
+// GetPipelineResourceID fetches resource ID for given full repository name
+func GetPipelineResourceID(client *jfroghttpclient.JfrogHttpClient, apiURL, repoName string, httpDetails httputils.HttpClientDetails) (int, bool, error) {
+	queryParams := make(map[string]string, 0)
+
+	uriVal, errURL := constructPipelinesURL(queryParams, apiURL, pipelineResources)
+	if errURL != nil {
+		return 0, false, errURL
+	}
+
+	resp, body, _, err := client.SendGet(uriVal, true, &httpDetails)
+	if err != nil {
+		return 0, false, errorutils.CheckError(err)
+	}
+	// Response Analysis
+	if err = errorutils.CheckResponseStatusWithBody(resp, body, http.StatusOK); err != nil {
+		return 0, false, err
+	}
+	if resp.StatusCode == http.StatusOK {
+		log.Debug("received resource id")
```

Comment:
This debug log meeaage is unclear and missing some context. Let's improve it, so that it is made clearer, or remove it altogether.